wd_helper.au3

```diff
@@ -991,9 +991,11 @@ EndFunc   ;==>_WD_ElementOptionSelect
 ;                  - $_WD_ERROR_InvalidDataType
 ;                  - $_WD_ERROR_InvalidExpression
 ;                  - $_WD_ERROR_InvalidArgue
+;                  - $_WD_ERROR_ElementIssue
+;                  - $_WD_ERROR_NoMatch
 ; Author ........: Danp2
 ; Modified ......: mLipok
-; Remarks .......: If no option is selected, SELECTEDINDEX will return -1
+; Remarks .......: If no option is selected, SELECTEDINDEX will return -1. Using MULTISELECT on not multiple able <select> element sets $_WD_ERROR_ElementIssue.
```

Comment:
I don't think this change is needed for Remarks. We don't explain each of the other possible errors codes, so why this one?

Reply:
Please take a look on: [remarks reverted](https://github.com/Danp2/au3WebDriver/pull/367/commits/e359cc047c19b8b20ab1bed82a424bd5eb02130c)